frontend/src/components/Calendar/ClickableDateContainer.tsx

```diff
@@ -4,22 +4,21 @@ import { useDispatch, useSelector } from 'react-redux'
 import { useHistory } from 'react-router-dom'
 import { setClickedDate } from '../../store/calendar/actions'
 import { RootState } from '../../store/types'
-import { PATHS } from '../../routes/Routes'
 
-const DateContainer = styled.div<{ selected?: boolean; currentDate?: boolean }>`
+const DateContainer = styled.div<{ selected?: boolean; currentDate?: boolean; overlay?: boolean }>`
```

Comment:
what is overlay? if the div can only be one of these states, consider combining them, with the type `'selected' | 'currentDate' | 'overlay (maybe think of a better word?)'`

Reply:
is overlay disabled?

Reply:
yes